consent-ws/src/main/java/org/broadinstitute/consent/http/models/Election.java

```diff
@@ -77,6 +77,20 @@ public Election(Integer electionId, String electionType,
         this.dataSetId = dataSetId;
     }
 
+    public Election(Integer electionId, String electionType,
+                    String status, Date createDate,
+                    String referenceId, Date lastUpdate, Boolean finalAccessVote, Integer dataSetId, Boolean archived) {
+        this.electionId = electionId;
+        this.electionType = electionType;
+        this.status = status;
+        this.createDate = createDate;
+        this.referenceId = referenceId;
+        this.lastUpdate = lastUpdate;
+        this.finalAccessVote = finalAccessVote;
+        this.dataSetId = dataSetId;
+        this.archived = archived;
+    }
+
```

Comment:
No feedback for this constructor, but I did notice that these election fields are not defined in `api-docs.yam;` It would be nice to add them at some point, perhaps a new tech debt story:

```
    private String dulName;
    private Boolean archived;
    private Integer version;
```

Reply:
These fields are now available on `api-docs.yaml`.